ui-tests/pages/authenticationPage.page.js

```diff
@@ -8,6 +9,7 @@ class authenticationPage {
     get landingPageWelcomeMessage() { return $('//h1[text()="Welcome"]') }
     get selectYes() { return $('//input[@value="Yes"]') }
     get logOutButton() { return $('//div//button[text()="Logout"]') }
+    get localeLogOutButton() { return $('//div//button[text()="xLogoutx"]') }
```

Comment:
Would a parameter in the step definition to cope better with the changes in text? A step like "The label for the logout button reads {string}"

Reply:
Done, thanks for suggesting :)